LightBulb.PlatformInterop/DeviceContext.cs

```diff
@@ -48,10 +53,46 @@ public void SetGamma(double redMultiplier, double greenMultiplier, double blueMu
         SetGammaRamp(ramp);
     }
 
+    public void SetBrightness(double brightness)
```

Comment:
I would move the brightness control to the `Monitor` class and have the `GammaService` call that directly. The `Monitor` class is already a wrapper around an `hMonitor` handle,  which is the same as your `PhysicalMonitor`, if I understand correctly.

Reply:
From the remarks:
> A single HMONITOR handle can be associated with more than one physical monitor. This function returns a handle and a text description for each physical monitor.
 When you are done using the monitor handles, close them by passing the pPhysicalMonitorArray array to the [DestroyPhysicalMonitors](https://learn.microsoft.com/en-us/windows/desktop/api/physicalmonitorenumerationapi/nf-physicalmonitorenumerationapi-destroyphysicalmonitors) function.